gtk4/Gir.toml

```diff
@@ -2138,10 +2150,24 @@ manual_traits = ["WidgetExtManual"]
         const = true
     [[object.function]]
     name = "get_name"
-    rename = "get_widget_name"
+    rename = "widget_name"
     [[object.function]]
     name = "set_name"
     rename = "set_widget_name"
+    [[object.function]]
+    name = "get_sensitive"
+    # Avoid clash with existing `is_sensitive` with a different meaning.
+    bypass_auto_rename = true
+    [[object.function]]
+    name = "get_visible"
+    # Avoid clash with existing `is_visible` with a different meaning.
+    bypass_auto_rename = true
+    [[object.function]]
+    name = "get_hexpand"
+    rename = "does_hexpand"
+    [[object.function]]
+    name = "get_vexpand"
+    rename = "does_vexpand"
```

Comment:
These are getters return `bool` values, so I used a conjugated verb. It didn't seem correct to use the form `widget.is_vexpand()`, so I used `widget.does_vexpand()`, but we might use `widget.vexpands()`. More compact, but less obvious too read maybe.

What do you think?

Reply:
I don't like the renamed variants, probably not renaming them is better than the `does_[h|v]expand` variants I find. 

Although the `vexpands`/`hexpands` looks much better to me to me.

Reply:
> the vexpands/hexpands looks much better to me to me

Yes, I prefer this too.